index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portifolio</title>
+    <link rel="stylesheet" href="index.css">
+  
+  </head>
+
+<body>
+    <header id="header">
+        <img id="header-img"
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" />
+        <nav id="nav-bar">
+            <ul>
+                <li>
+                    <a class="nav-link" href="#section1">field1</a>
+                </li>
+                <li>
+                    <a class="nav-link" href="#section2">field2</a>
+                </li>
+                <li>
+                    <a class="nav-link" href="#section3">field3</a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <video id="video" width="320" height="240" controls>
+        <source src="https://www.youtube.com/watch?v=y8Yv4pnO7qc" type="video/mp4" />
+        Your browser does not support the video tag.
+    </video>
+    <section id="section1">
+        <h1>Section 1</h1>
+        <p>This is a paragraph</p>
+        <p>This is a paragraph</p>
+        <p>This is a paragraph</p>
+        <p>This is a paragraph</p>
+        <p>This is a paragraph</p>
+        <p>This is a paragraph</p>
+    </section>
+    <section id="section2">
+        <h1>Section 2</h1>
```

Comment:
- I would like to advice you that not to use more than one h1 in your project. Instead use more h2, h3 h4... headers more than once